server/src/main/java/com/unosquare/carmigo/service/PlatformUserService.java

```diff
@@ -54,13 +53,16 @@ public GrabPlatformUserDTO createPlatformUser(final CreatePlatformUserDTO create
         return modelMapper.map(platformUserRepository.save(platformUser), GrabPlatformUserDTO.class);
     }
 
-    public GrabPlatformUserDTO updatePlatformUser(final int id, final JsonPatch patch)
+    public GrabPlatformUserDTO patchPlatformUser(final int id, final JsonPatch patch)
     {
-        final PlatformUser targetPlatformUser = modelMapper.map(getPlatformUserById(id), PlatformUser.class);
+        final GrabPlatformUserDTO grabPlatformUserDTO = modelMapper.map(
+                findPlatformUserById(id), GrabPlatformUserDTO.class);
         try {
-            final PlatformUser patchedPlatformUser = applyPatchToPlatformUser(patch, targetPlatformUser);
+            final JsonNode platformUserNode = patch.apply(
```

Comment:
try and not use try catches and let your exception handler catch anything that is unexpected.

Reply:
you have a try / catch here.  Catch exceptions in the global exception handler.